Update to note a detail about multiple web apps

milope opened the pull request:
As stated in the change: If multiple web apps are added to the same backend address pool and `-PickHostNameFromBackendAddress` is used, you risk having the wrong web app serving the content. Example, if the backend address pool has both www.contoso.com and www.adventureworks.com, when using `-PickHostNamefromBackendHttpSettings` can result in www.adventureworks.com being served by the www.contoso.com web app instead as we randomly were assigned to www.contoso.com as the backend address server instead of www.adventureworks.com. It is recommended to use multi-site HTTP listeners to host multiple web apps behind a single load balancer.

azurecla commented:
Hi __@ClarityQueen__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (v-diree). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

cjgronlund commented:
@ClarityQueen 
I'm closing this PR without accepting. You edits would normally be valid, but these are button selectors, so the shortened names used are necessary in context.

milope committed:
Update to note a detail about multiple web apps

As stated in the change: If multiple web apps are added to the same backend address pool and `-PickHostNameFromBackendAddress` is used, you risk having the wrong web app serving the content. Example, if the backend address pool has both www.contoso.com and www.adventureworks.com, when using `-PickHostNamefromBackendHttpSettings` can result in www.adventureworks.com being served by the www.contoso.com web app instead as we randomly were assigned to www.contoso.com as the backend address server instead of www.adventureworks.com. It is recommended to use multi-site HTTP listeners to host multiple web apps behind a single load balancer.
